core/proto/squareup/subzero/internal.proto

```diff
@@ -111,6 +117,8 @@ enum Result {
   NO_ROLLBACK_INVALID_FORMAT = 49;
   NO_ROLLBACK_INVALID_MAGIC = 50;
   NO_ROLLBACK_INVALID_VERSION = 51;
+  SIG_CHECK_FAILED = 52;
```

Comment:
nit: should we use `QRSIG` instead of `SIG`? there are different kind of signatures across the codebase (QR code signatures, and transaction signatures), so having a quick way to tell them apart might be helpful in the future.